extensions/wikia/AssetsManager/config.php

```diff
@@ -2492,3 +2492,11 @@
 		'//extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js',
 	]
 ];
+
+$config['cookiesyncer_js'] = [
```

Comment:
Is it concatenated with other packages or downloaded separately? If the latter, then I'd add the script to `oasis_shared_core_js` or something like that.

Reply:
@rogatty fix in bebadc82140